packages/utils/src/normalize.ts

```diff
@@ -24,18 +26,18 @@ type UnknownMaybeWithToJson = unknown & { toJSON?: () => string };
  * object in the normallized output..
  * @returns A normalized version of the object, or `"**non-serializable**"` if any errors are thrown during normalization.
  */
-export function normalize(input: unknown, depth: number = +Infinity, maxProperties: number = +Infinity): any {
+export function normalize(input: unknown, depth: number = +Infinity, maxProperties: number = +Infinity): unknown {
```

Comment:
Nice TS. 🙂

I tried it out, and it ended up getting complicated because it then required a similar (but not identical) type change for `visit()`... at which point it seemed like a big enough potential change that it would deserve its own PR.

> to help us in the future

If we were to do that, I'm curious, how do you see this helping our future selves?

As for the change to `unknown`, I did it here just because I generally always try to `s/any/unknown` where I can, because I am a slave to the linter. This is making me reconsider that choice, though, because it then required changes downstream that don't really have anything to do with the real point of this PR. On second thought, I think I'll hold off for now, until we can fix it the "right way," with an actual type as you suggest.

Reply:
> to help us in the future

By that I meant that it's basically always a good thing to narrow return types down as much as possible (callbacks are an exceptions to this). Having narrow return types means we can make just better assertions in other places in the code without always having to cast stuff. I'm thinking: If we already know what the thing we're looking at looks like, let's just type it.

But I'm fine with doing that typing stuff whenever :D